Lib/sqlite3/test/hooks.py

```diff
@@ -261,6 +272,21 @@ def trace(statement):
         cur.execute(queries[1])
         self.assertEqual(traced_statements, queries)
 
+    def test_trace_expanded_sql(self):
```

Comment:
Why aren't the tests from `test_trace_too_much_expanded_sql` included? Does the framework for those not exist in 3.10?

Reply:
Yep, see https://github.com/python/cpython/pull/92147#issuecomment-1115002300